Model/Menu/NodeRepository.php

```diff
@@ -29,6 +38,11 @@ public function __construct(
         $this->searchResultsFactory = $searchResultsFactory;
     }
 
+    /**
+     * @param NodeInterface $object
+     * @return NodeInterface
+     * @throws CouldNotSaveException
+     */
```

Comment:
I would use `@inheritdoc` instead (I see it's used already in Menu.php model class, so let's keep the same format)

Reply:
Added changes for all methods